test/data/edge/commit_body.json

```diff
@@ -0,0 +1,21 @@
+{
+    "distribution": "rhel-84",
```

Comment:
More distro dependent stuff. I'm not sure if it will break once we switch to 8.5 or if we simply have to maintain the distro references in several places. I don't like either case, IDK what to do about it.

Reply:
This PR is supposed to run in RHEL8.4.
But thanks for your comment, I will remove commit_body.json and installer_body.json, instead I will use variable $DISTRO and generate these files in test script.

Reply:
FTR we're very close to switching to 8.5 for the unreleased versions and the `rhel-8` runner is still 8.3.